webservice/routes/api.php

```diff
@@ -3,11 +3,13 @@
 Route::group([
     'middleware' => ['cors', 'api'],
 ], function () {
-    Route::post('/login', 'LoginController@login');
+    Route::post('/auth/issue', 'AuthController@issueToken');
```

Comment:
May I jump in to give my 10 cents?

I think *issue* would make sense if the route was */token/issue*. The way it is it's misleading.